libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -1013,7 +1013,9 @@ void NavEKF3::UpdateFilter(void)
     }
 
     // align position of inactive sources to ahrs
-    sources.align_inactive_sources();
+    for (uint8_t i=0; i<num_cores; i++) {
+        sources.align_inactive_sources(i);
```

Comment:
it doesn't matter functionally but we might as well move this loop into the align_inactive_sources() method